scripts/build/auroTemplateFiller.mjs

```diff
@@ -49,8 +49,8 @@ export class AuroTemplateFiller {
 
     const pName = parsedPackageJson.name;
     const pVersion = parsedPackageJson.version;
-    const pdtVersion = parsedPackageJson.peerDependencies[AuroTemplateFiller.designTokenPackage].substring(1);
-    const wcssVersion = parsedPackageJson.peerDependencies[AuroTemplateFiller.webCoreStylesheetsPackage].substring(1);
+    const pdtVersion = parsedPackageJson.peerDependencies?.[AuroTemplateFiller.designTokenPackage].substring(1) ?? '';
```

Comment:
This is a super minor thing and can be left as is. Thoughts on more verbose variable names? I was really only able to understand what the variable was because how it was being set had some clues. Later on when trying to read parts of this file where the variable is being used it might be hard to understand without looking it up.

Reply:
oh yeah I'm not a huge fan of the variable names, maybe this would have been a good time to update those instead of copy-pasting from the old code 🤔

Reply:
Your call 👍 I am on team rename.